src/commands/compose/compose.ts

```diff
@@ -38,26 +39,41 @@ async function compose(context: IActionContext, commands: ('up' | 'down')[], mes
 
     for (const command of commands) {
         if (selectedItems.length === 0) {
-            const terminalCommand = await selectComposeCommand(
+            // Get the initial command
```

Comment:
Given these two blocks are nearly identical, is there a way to consolidate them into a single block?  For example, wrap the no-selected-items case into a "dummy" selected item so that you can just use the selected items branch?